types/es-to-primitive/ts3.2/es2015.d.ts

```diff
@@ -1,5 +1,5 @@
 declare function ToPrimitive(
     input: unknown,
     hint?: typeof String | typeof Number,
-): string | number | bigint | boolean | symbol | null | undefined;
```

Comment:
wouldn't we want to keep bigint in the TS 3.2 dir?

Reply:
Yes, I messed up es-to-primitive -- I correctly moved the contents of the  ts3.2 dir, but the 3.1 dir should have stayed the same. Instead I moved it to ts3.2.

(The root files are just deleted because they apply only to 3.0 and below.)